packages/updatePrescriptionStatus/src/utils/responses.ts

```diff
@@ -0,0 +1,121 @@
+import {Bundle, BundleEntry} from "fhir/r4"
+
+export function bundleWrap(entries: Array<BundleEntry>): Bundle {
+  return {
+    resourceType: "Bundle",
+    type: "transaction-response",
+    entry: entries
+  }
+}
+
+export function badRequest(diagnostics: string, fullUrl: string | undefined = undefined): BundleEntry {
+  const bundleEntry: BundleEntry = {
+    response: {
+      status: "400 Bad Request",
+      outcome: {
+        resourceType: "OperationOutcome",
+        meta: {
+          lastUpdated: new Date().toISOString()
+        },
+        issue: [
+          {
+            code: "processing",
+            severity: "error",
+            diagnostics: diagnostics,
+            details: {
+              coding: [
+                {
+                  system: "https://fhir.nhs.uk/CodeSystem/Spine-ErrorOrWarningCode",
+                  code: "INVALID_VALUE",
+                  display: "Invalid value"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    }
+  }
+  if (fullUrl) {
+    bundleEntry.fullUrl = fullUrl
+  }
+  return bundleEntry
+}
+
+export function accepted(fullUrl: string): BundleEntry {
+  return {
+    fullUrl: fullUrl,
+    response: {
+      status: "200 Accepted",
+      outcome: {
+        resourceType: "OperationOutcome",
+        meta: {
+          lastUpdated: new Date().toISOString()
+        },
+        issue: [
+          {
+            code: "informational",
+            severity: "information",
+            diagnostics: "No issues detected during validation."
```

Comment:
We should explain here that the data has not been committed due to issues in the other entries.

Reply:
Yep, makes sense. Have changed.